packages/react-native-session-replay/DatadogSDKReactNativeSessionReplay.podspec

```diff
@@ -19,9 +19,10 @@ Pod::Spec.new do |s|
   s.dependency "React-Core"
 
   # /!\ Remember to keep the version in sync with DatadogSDKReactNative.podspec
-  s.dependency 'DatadogSessionReplay', '~> 2.4.0'
+  s.dependency 'DatadogSessionReplay', '~> 2.5.0'
   
   s.test_spec 'Tests' do |test_spec|
+    test_spec.dependency "React-RCTText"
```

Comment:
I did not really understand why this was only needed for tests. If I'm not adding this the tests fail with a `Undefined symbols for architecture arm64: "_OBJC_CLASS_$_RCTTextView"` error.
However running an app works fine. I assumed this was some RN black magic, but I'd be happy to get a second opinion on this.

As `React-RCTText` does not declare modules, pod install will fail unless `use_modular_headers!` is used in the podfile, so I only add the dependency to the test spec so our users are not impacted.